cumulus/parachains/integration-tests/emulated/chains/parachains/assets/asset-hub-rococo/src/genesis.rs

```diff
@@ -20,17 +20,26 @@ use sp_keyring::Sr25519Keyring as Keyring;
 
 // Cumulus
 use emulated_integration_tests_common::{
-	accounts, build_genesis_storage, collators, PenpalASiblingSovereignAccount,
-	PenpalATeleportableAssetLocation, PenpalBSiblingSovereignAccount,
-	PenpalBTeleportableAssetLocation, RESERVABLE_ASSET_ID, SAFE_XCM_VERSION, USDT_ID,
+	accounts, build_genesis_storage, collators, xcm_emulator::ConvertLocation,
+	PenpalASiblingSovereignAccount, PenpalATeleportableAssetLocation,
+	PenpalBSiblingSovereignAccount, PenpalBTeleportableAssetLocation, RESERVABLE_ASSET_ID,
+	SAFE_XCM_VERSION, USDT_ID,
 };
 use parachains_common::{AccountId, Balance};
+use snowbridge_inbound_queue_primitives::EthereumLocationsConverterFor;
```

Comment:
DEPRECATED in favor of [xcm_builder::ExternalConsensusLocationsConverterFor]

Reply:
https://github.com/vgeddes/polkadot-sdk/pull/58/commits/63f1d31e463fe3de08e8900b1ce4790ce0106a16